BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+# GraphQL.NET Bounty Program
+
+The GraphQL.NET team has worked to create a program where community developers can work on specific new features
+of GraphQL.NET for compensation. These features are determined by GraphQL.NET maintainers to have the
+potential for high-impact improvement to the product. Design proposals that would fulfill a bounty may
+be made either by individuals or teams of developers, but the final payout will be made to only one stakeholder.
+
+
+## Bounty Process Flowchart
+
+1. The GraphQL.NET team publishes a Request For Proposals (RFP) describing a feature that the team has decided
+   should be prioritized, or a bug that is in need of being researched and fixed.
+    * Initial Bounty Status set to **Accepting Proposals**
+2. Developers write proposals for how they would implement the feature or fix the bug described in the RFP.
+3. The GraphQL.NET team works with developers to clarify and refine their proposals.
+4. A proposal is accepted. The developer(s) responsible for the proposal work to implement the feature or
+   bug fix according to their proposal.
+    * Bounty status changed to **In Progress**
+5. Pull requests are submitted to implement the proposal by the developers.
+    * Bounty status changed to **Under Review**
+6. Pull requests are reviewed by project maintainers, and the pull request developers make any changes to the
+   pull requests accordingly.
+7. Pull requests are merged into the master or develop branch.
+8. Developer(s) submit invoice to the [GraphQL.NET Open Collective](https://opencollective.com/graphql-net)
+   for bounty amount, confirmed by GraphQL.NET team members.
+    * Bounty status changed to **Claimed**
+9. Payment disbursed to developer(s) by the Open Source Collective from GraphQL.NET funds.
+
+
+## How to Claim a Bounty
```

Comment:
I do not understand how this section differs from `# Bounty Process Flowchart`. Yes, they differ of course, but both describe about the same.

Reply:
They are not much different.  Just written from different points of view.  Seemed fine to me, since we don't have much else to say.

Reply:
I am inclined to agree. Over time, we may find perfect formulations and descriptions.